src/test/java/lottogame/domain/lotto/LottoTest.java

```diff
@@ -0,0 +1,43 @@
+package lottogame.domain.lotto;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class LottoTest {
+    private Lotto lotto;
+
+    @BeforeEach
+    void setUp() {
+        List<Integer> randomValues = Arrays.asList(new Integer[]{8, 21, 23, 41, 42, 43});
```

Comment:
```suggestion
        List<Integer> randomValues = Arrays.asList(8, 21, 23, 41, 42, 43);
```
위와 같이 사용할 수도 있어요 :)
그리고, 전체적으로 배열보단 Collection을 사용하는걸 권장합니다!

Reply:
모든 코드 내에 있는 배열들을 지우고 Collection으로 변경하였습니다 😊😊👍